source/adapters/level_zero/event.cpp

```diff
@@ -15,18 +15,17 @@
 
 #include "common.hpp"
 #include "event.hpp"
+#include "logger/ur_logger.hpp"
 #include "ur_level_zero.hpp"
 
 void printZeEventList(const _ur_ze_event_list_t &UrZeEventList) {
-  if (UrL0Debug & UR_L0_DEBUG_BASIC) {
-    urPrint("  NumEventsInWaitList %d:", UrZeEventList.Length);
+  std::stringstream ss;
+  ss << "  NumEventsInWaitList" << UrZeEventList.Length << ":";
 
-    for (uint32_t I = 0; I < UrZeEventList.Length; I++) {
-      urPrint(" %#llx", ur_cast<std::uintptr_t>(UrZeEventList.ZeEventList[I]));
-    }
-
-    urPrint("\n");
```

Comment:
previously you had a new line after each entry, now you have it all printed at once (with spaces between) - is this expected?

Reply:
There was only one new line

Reply:
haha, mea culpa, I'm blind, apparently 😉  // the second `}` (from `if`) got me, I think